src/index.html

```diff
@@ -11,7 +11,196 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+
+      <fieldset class="form__fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form__field">
+          Surname:
+
+          <input 
+            type="text" 
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+
+          <input 
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form__field">
          Name:
          <input 
            type="text"
            name="name"
            required
            autocomplete="off"
          >
        </label>
```

Dont add blank lines between parent and child blocks

FIx all such cases